ajax/libs/print.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "print.js",
+  "filename": "print.min.js",
+  "homepage": "http://printjs.crabbly.com",
+  "author": "Rodrigo Vieira <rodrigo@crabbly.com>",
+  "description": "A tiny javascript library to help printing from the web.",
+  "version": "1.0.24",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crabbly/Print.js.git"
+  },
+  "keywords": [
+    "javascript",
+    "print.js",
+    "printjs"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/crabbly/Print.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
We still can add the origin css files (non-minified) which under `src` folder.
Would you please add this part in auto-update config?
It will be a great work.

Many thanks.

```js
      {
        "basePath": "src",
        "files": [
          "css/print*"
        ]
      }
```